webapp/src/pages/ResponsePage/ResponsePage.js

```diff
@@ -0,0 +1,210 @@
+
+import React, { Component } from 'react'
+import ReactTable from 'react-table';
+import { NavLink } from 'react-router-dom';
+import "react-table/react-table.css";
+import { withTranslation } from 'react-i18next';
+import './ResponsePage.css'
+import { applicationFormService } from '../../services/applicationForm/applicationForm.service'
+import { fetchResponse } from '../../services/ResponsePage/ResponsePage'
+
+class ResponsePage extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            applicationForm: null,
+        }
+    };
+
+
+
+    componentDidMount() {
+        this.fetchForm()
+        this.fetchData()
+    }
+
+
+
+    // Fetch Form
+    fetchForm() {
+        applicationFormService.getForEvent(this.props.event.id).then(response => {
+            this.setState({
+                applicationForm: response.formSpec
+            })
+        })
+    }
+
+
+
+    // Fetch Data
+    fetchData() {
+        let params = {
+            id: this.props.match.params.id,
+            eventKey: this.props.match.params.eventKey,
+        }
+
+        fetchResponse(params).then(response => {
+            this.setState({
+                applicationData: response
+            })
+        })
+    }
+
+
+
+    // Go Back
+    goBack() {
+        this.props.history.goBack();
+    }
+
+
+
+    // Generate Applciation Status
+    applicationStatus() {
+        const data = this.state.applicationData;
+
+        if (data) {
+            let unsubmitted = !data.is_submitted && !data.is_withdrawn;
+            let submitted = data.is_submitted;
+            let withdrawn = data.is_withdrawn;
+
+            if (unsubmitted) {
+                return ["unsubmitted" + " " + data.started_timestamp]
+            }
+            if (submitted) {
+                return ["submitted" + " " + data.submitted_timestamp]
+            }
+            if (withdrawn) {
+                return ["withdrawn" + " " + data.withdrawn_timestamp]
+            }
+        }
+    }
+
+
+
+    // Render Sections
+    renderSections() {
+        const applicationForm = this.state.applicationForm;
+        const applicationData = this.state.applicationData;
+        let html = [];
+        // Translation
+        const t = this.props.t;
+
+        // main function
+        if (applicationForm && applicationData) {
+            applicationForm.sections.forEach(section => {
+                html.push(<div className="section">
+                    { /*Heading*/}
+                    <div className="flex baseline"><h3>{section.name}</h3><label>{t('Section')}</label></div>
```

Comment:
Please remove this "Section" label - anyone using this page will already know it's a section